.travis.yml

```diff
@@ -45,64 +50,119 @@ sudo: false
 
 # Compilation dependencies
 addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - git-core
-      - build-essential
-      - g++-5
-      - libboost-all-dev
-      - libhdf5-serial-dev
-      - libfftw3-dev
-      - python-dev
-      - python3-dev
-      - liblapack-dev
-      - libxml2-dev
-      - libxslt-dev
-      - libarmadillo-dev
-      - libace-dev
-      - libgtest-dev
-      - libplplot-dev
-      - root-system-bin
-      - swig
+ apt:
+  sources:
+   - ubuntu-toolchain-r-test
+  packages:
+   - git-core
+   - build-essential
+   - g++-5
+   - libboost-all-dev
+   - libhdf5-serial-dev
+   - libfftw3-dev
+   - python-dev
+   - python3-dev
+   - python-tk
```

Comment:
why include this? We don't need it as far as I know

Reply:
we definitely need `-dev` and `-tk`, arguably not for both `python` and `python3` (depending on the build matrix) but

1. it's neater to put both here rather than require `sudo` to `apt-get` the appropriate version
2. it's a quick download/install
3. it doesn't affect `osx`

What about the other dependencies, such as `root-system-bin`? Are they really all required? I just copied from the `SuperBuild`.

Reply:
agree to install both python2 and 3 here. still mystified about `-tk` as I've never had to install this before (maybe it was there). 

yes, get rid of `root-system-bin`. Could be used to enable ROOT support for STIR, but then it'd need other packages as well (and is sure to let to other compilation problems, so we won't do this now).

Reply:
ok, building without `root-system-bin` now. `python-tk` is needed (https://travis-ci.org/CCPPETMR/SIRF/jobs/315938695) and would have been installed back when we had `language: python`. We need `language: cpp` now in order to get `ccache` to work (which is evidently much more important than caching `pip`)

let me know if you think anything else can be removed. it's about 1:30min to apt-get things at the moment.